docker/centos7/Dockerfile

```diff
@@ -128,8 +130,25 @@ RUN source /opt/rh/devtoolset-8/enable && \
         -G Ninja \
         -DLLVM_INCLUDE_EXAMPLES=OFF \
         -DLLVM_INCLUDE_TESTS=OFF \
+        -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra;lld;lldb" \
+        -DLLVM_STATIC_LINK_CXX_STDLIB=ON \
+        ../llvm && \
+    cmake --build . && \
+    cmake --build . --target install && \
+    cd .. && \
+# It must be a clean directory, otherwise there may be undefined behavior.
+    mkdir build2 && \
```

Comment:
We should name it a bit more intuitive, maybe `build-libs`?

Reply:
done